witness/golang/omniwitness/usbarmory/internal/storage/slots/journal.go

```diff
@@ -102,12 +106,13 @@ func (e *entry) Size() int {
 // OpenJournal returns a new journal structure for interacting with a journal stored in the
 // [start, start+length) range of blocks accessible via dev.
 // Journal ranges should not overlap with one another, or corruption will almost certainly occur.
-func OpenJournal(dev BlockReaderWriter, start, length uint) (*Journal, error) {
+func OpenJournal(dev BlockReaderWriter, start, length uint, h Hasher) (*Journal, error) {
 	j := &Journal{
 		dev:          dev,
 		start:        start,
 		length:       length,
 		maxDataBytes: (length*dev.BlockSize())/minEntries - entryHeaderSize,
+		hasher:       h,
```

Comment:
nit: probably subjective style, but while I prefer `hasher` for the field name from a lifetime of doing OO, from an applied perspective, `hash` is a better name because it's a straight function. I think `hash` looks better at the point of use.

Reply:
renamed :)